db/db.json

```diff
@@ -1,18 +1,34 @@
 {
   "contacts": [
     {
-      "id": 1,
-      "firstName": "John",
-      "lastName": "Carmack",
-      "street": "10 Keen Street",
-      "city": "London"
+      "firstName": "Jonathan",
+      "lastName": "Phython",
+      "street": "C++",
+      "city": "Code Wars",
+      "email": "sdadsdas@sdds.com",
+      "linkedIn": "FreeCode",
+      "twitter": "Camp",
+      "id": 2
     },
     {
-      "id": 2,
-      "firstName": "Grace",
-      "lastName": "Hopper",
-      "street": "32 Deebug Road",
-      "city": "London"
+      "firstName": "Kiran",
+      "lastName": "Script",
+      "street": "Highway JSX",
+      "city": "Html Borough",
+      "email": "sdadsdas@sdds.com",
+      "linkedIn": "MDN",
+      "twitter": "VSCode",
+      "id": 3
+    },
+    {
+      "firstName": "Jiggly",
```

Comment:
Jigglypuff was a bit of a diva in the anime. I really liked Wobbuffet - he had the most funny moments